pychromecast/socket_client.py

```diff
@@ -231,8 +232,10 @@ def initialize_connection(self):
 
                 self._report_connection_status(
                     ConnectionStatus(CONNECTION_STATUS_FAILED, None))
-                self.logger.exception("Failed to connect, retrying in %fs",
-                                      self.retry_wait)
+                retry_log_fun("Failed to connect, retrying in %fs",
+                              self.retry_wait)
+                retry_log_fun = self.logger.debug
```

Comment:
Nah I think it's fine. DIdn't realize it was inside the `initalize_connection` method.